tests/test_repair.py

```diff
@@ -331,6 +331,14 @@ def test_repair_no_intact_axon():
         test_module.repair(filename, outfilename, axons=[filename])
 
 
+def test_repair_no_trunk():
+    '''Test repair when the morph has oblique sections but no trunk'''
+    filename = DATA_PATH / 'Fluo42_right.h5'
+    with TemporaryDirectory('test-no-trunk') as tmp_folder:
+        outfilename = Path(tmp_folder, 'out.asc')
+        test_module.repair(filename, outfilename, legacy_detection=True)
```

Comment:
don't we need to assert here the number of expected obliques?

Reply:
Do you mean before calling `repair()` ?

Reply:
I mean after. Just as in `test_repair_axon`. How can we verify that the previously missed obliques were counted this time?

Reply:
Ah, no they were not simply missed. The code was crashing ! :)
So just the fact that the code runs is enough to cover my change

Reply:
ah, ok then.